docs/docker.md

```diff
@@ -1,134 +1,151 @@
 
+For more information see [miner configuration documentation](configuration#miner-configuration).
+
+## Testnet
+
+The minimal configuration to join the testnet needs a list of seed peers and network identifier:
+
+```yaml
+---
+peers:
+    # UAT
+    - aenode://pp_QU9CvhAQH56a2kA15tCnWPRJ2srMJW8ZmfbbFTAy7eG4o16Bf@52.10.46.160:3015
+    - aenode://pp_2vhFb3HtHd1S7ynbpbFnEdph1tnDXFSfu4NGtq46S2eM5HCdbC@18.195.109.60:3015
+    - aenode://pp_27xmgQ4N1E3QwHyoutLtZsHW5DSW4zneQJ3CxT5JbUejxtFuAu@13.250.162.250:3015
+    - aenode://pp_nt5N7fwae3DW8Mqk4kxkGAnbykRDpEZq9dzzianiMMPo4fJV7@18.130.148.7:3015
+
+fork_management:
+    network_id: ae_uat
+
+```
+
+Assuming your configuration file is located at `~/.aeternity/myepoch.yaml` on the host machine:
 
 ```bash
-docker run -d -p 3013:3013 \
-    -v ~/.aeternity/myepoch.yaml:/home/epoch/myepoch.yaml \
-    -e EPOCH_CONFIG=/home/epoch/myepoch.yaml \
-    --hostname node0 \
-    -v ~/.aeternity/db:/home/epoch/node/data/mnesia \
-    -v ~/.aeternity/keys:/home/epoch/node/keys \
-    aeternity/epoch
+docker run -P -v ~/.aeternity/myepoch.yaml:/home/epoch/.epoch/epoch/epoch.yaml aeternity/epoch
 ```
 
-**Note: make sure `hostname` option is set when reusing the mnesia data directory**
+You should see the console output of the running node and a lot of information related to syncing with the network.
+
+See [how to persist the blockchain data](#persisting-data) and [how to enable mining](#mining) above.
+
 
 ## Localnet
 
-Small local network (*not* connected to testnet) can be created with `docker-compose`.
+Small local network (*not* connected to public networks) can be created with `docker-compose`.
 It runs three nodes using the `mean15-generic` miner (fastest generic miner) and a proxy server to allow CORS.
+As the beneficiary key-pair is publicly available, this setup should *not* be connected to public networks.
 
-All local network nodes have `ak_25eTK8PaiLpREqBkP3yDNWJAwXjWSR8tbn3zu8SXaNx824A1AJ` set as node beneficiary (for more details on beneficiary see [configuration documentation](configuration.md#beneficiary-account)).
-Public-private keypair of `ak_25eTK8PaiLpREqBkP3yDNWJAwXjWSR8tbn3zu8SXaNx824A1AJ` beneficiary can be found [here](/docker/keys/beneficiary): as the private key is publicly available, this setup must *not* be connected on the live network.
-Base58Check-encoded form of private key, which matches above beneficiary public key: `7eFs4YMo1bXS7Z5sFvmgb2WpJrGpoWFqVZY4mb4o99c7aAbxdTko1tV3toCbumDRpbSpSceDqTYLRnE5Utr869KhWshSo`.
+All local network nodes are configured with the same beneficiary account (for more details on beneficiary see [configuration documentation](configuration.md#beneficiary-account)):
+- public key: ak_twR4h7dEcUtc2iSEDv8kB7UFJJDGiEDQCXr85C3fYF8FdVdyo
+- private key secret: `secret`
+- key-pair binaries can be found [here](/docker/keys/beneficiary)
```

Comment:
This will be dead link in release package.
Shall this docker.md be removed from release package?

Reply:
Maybe .. no strong opinion the link was there, I just kept it

Reply:
removed in 46655314